CHANGES.rst

```diff
@@ -6,6 +6,9 @@ There's a frood who really knows where his towel is.
 1.0a13 (unreleased)
 ^^^^^^^^^^^^^^^^^^^
 
+- Implement CSRF protection (closes `590`_).
```

Comment:
plone.protect version is what motivates installing plone4.csrffixes or not. See https://github.com/plonegovbr/brasil.gov.portal/issues/241#issuecomment-162065866 and https://github.com/plone/Products.CMFPlone/commit/07ec9bc7aa06bbb8adb11d7d69e24b1f3a080d08#commitcomment-16457187

Would it be the case to warn the user to add plone4.csrffixes if it's using 4.3.x and plone.protect >= 3.016, like you warned about plone.app.stagingbehavior?